synapse/cores/common.py

```diff
@@ -907,12 +904,10 @@ def delTufoTag(self, tufo, tag):
         props = s_tags.getTufoSubs(tufo,tag)
```

Comment:
maybe same kinda deal here if getTufoSubs already has a string for the tag only...